Mathlib/Geometry/Manifold/ContMDiff/Defs.lean

```diff
@@ -57,17 +57,18 @@ variable {𝕜 : Type*} [NontriviallyNormedField 𝕜]
   -- declare a smooth manifold `M` over the pair `(E, H)`.
   {E : Type*}
   [NormedAddCommGroup E] [NormedSpace 𝕜 E] {H : Type*} [TopologicalSpace H]
-  (I : ModelWithCorners 𝕜 E H) {M : Type*} [TopologicalSpace M] [ChartedSpace H M]
+  (I : ModelWithCorners 𝕜 E H) {M : Type*} [TopologicalSpace M] [CS : ChartedSpace H M]
```

Comment:
Same here, these can be removed, right?

Reply:
Needed for `ContMDiffOn` and `contMDiffOn_iff` in `contMDiff_openEmbedding`.